spec/results-model-spec.js

```diff
@@ -33,6 +35,10 @@ describe("ResultsModel", () => {
       expect(resultsModel.getPathCount()).toBe(1);
       expect(resultsModel.getMatchCount()).toBe(6);
       expect(resultsModel.getPaths()).toEqual([editor.getPath()]);
+      if (parseFloat(atom.getVersion()) >= 1.17) {
```

Comment:
For the [previous PR](https://github.com/atom/find-and-replace/pull/847/files#diff-eb24277759b68118e9e07ef629d193e8R250) the following was recommended to compare the Atom version: https://github.com/atom/find-and-replace/pull/847#issuecomment-293036002 Looking back the interpretation as a float is just wrong. E.g. version 1.2 would satisfy this check. Since I need this for the specs too I wanted to ask what I should use instead. Is there a recommended way to compare the version number? Should I pull in `semver` as  a direct dependency of this package?